autoload/lsp/ui/vim/folding.vim

```diff
@@ -31,7 +31,7 @@ function! s:set_textprops(buf) abort
     silent! call prop_type_add(s:textprop_name, {'bufnr': a:buf})
 
     " First, clear all markers from the previous run
-    call prop_remove({'type': s:textprop_name, 'bufnr': a:buf})
+    call prop_remove({'type': s:textprop_name, 'bufnr': a:buf}, 1, line('$'))
```

Comment:
@studiome Just wondering, `line('$')` returns the number of lines in the current buffer, however, shouldn't we make sure to get the number of lines in the buffer `a:buf` instead? I remember some problems with not specifying `a:buf` when I implemented folding.

Reply:
Using a:buf instead of ‘$’ occurs error “Invalid Range”. Umm...

Reply:
Yeah, `line` takes a line expr and a window id (optionally), so you still need the `'$'`. Unfortunately, it doesn't take a buffer id, and I don't know a function that does off the top of my head, so I guess we can leave it until it causes problems.